torchci/components/WorkflowBox.tsx

```diff
@@ -114,10 +115,50 @@ export default function WorkflowBox({
   const [artifactsToShow, setArtifactsToShow] = useState(new Set<string>());
   const groupedArtifacts = groupArtifacts(jobs, artifacts);
 
+  const [searchString, setSearchString] = useState("");
+  const [searchRes, setSearchRes] = useState<{
+    results: Map<string, [number[], string[]] | undefined>;
+    info: undefined | string;
+  }>({
+    results: new Map(),
+    info: undefined,
+  });
+  useEffect(() => {
+    getSearchRes(jobs, searchString, setSearchRes);
+  }, [searchString]);
+
   return (
     <div id={anchorName} className={workflowClass}>
       <h3>{workflowName}</h3>
-      <h4>Job Status</h4>
+      <div>
+        <div
+          // Similar styling to an h4
+          style={{ float: "left", marginBottom: "1.33em", fontWeight: "bold" }}
+        >
+          Job Status
+        </div>
+        <div style={{ float: "right" }}>
+          <form
+            style={{ float: "right" }}
+            onSubmit={(e: React.FormEvent<HTMLFormElement>) => {
+              e.preventDefault();
+              // @ts-ignore
+              const searchString = e.target[0].value;
+              setSearchString(searchString);
+            }}
+          >
+            <input type="text" placeholder="Search raw logs"></input>
```

Comment:
Nit: Do you think we should make it clearer that regex is supported here, i.e. `Search logs with regex`? I didn't realize that until I read your code below using `RegExp`.

Reply:
Oops I forgot I put that there, changed the placeholder text to mention regex